text/0107-construct-library-module-lifecycle.md

```diff
@@ -0,0 +1,144 @@
+
+* **Stage 1 activities:**
+
+    * **Update GitHub tracking issue** - Summarize the primary use cases and required constructs as they are identified
+    * **Author RFCs [optional]** - Request for comment documents are created for large / complex designs that require explanation and robust discussion. RFCs are stored in a separate [GitHub repo](https://github.com/aws/aws-cdk-rfcs), and the module's tracking issue will include links to relevant RFCs.
+    * **Update Roadmap** - publicly indicate the module is work in progress
+        * If an RFC is written, place the module’s tracking issue on the CDK roadmap in the “researching” column to indicate we need customer feedback on the module’s design
+        * otherwise, place the module’s tracking issue on the CDK roadmap in the “Working on it“ column
+    * **Submit Pull Requests** - pull requests implementing the constructs are submitted, the API design reviewed (by module owner), and merged
+    * **Documentation** - update module’s README with explanations and code snippets that illustrate how to use new constructs as they are developed
+    * **Publish aws-xyz-experimental package** - deliver experimental constructs in their own package to clearly communicate module maturity. This package is tagged with Stability:Experimental
+    * **Observe** - monitor module utilization and GitHub issues filed against the experimental module
+        * feature requests and bugs filed against the experimental module are triaged, those that we decide to include in the module are linked in the tracking issue to indicate they are in scope
+    * **Repeat** - These activities are repeated until the module owner decides the module is feature complete and the API is ergonomic. At the end of the experimental stage, the construct library module’s GitHub tracking issue will have a description of the service, a list of targeted use cases, and a list of L2 constructs required to satisfy the use cases.
+* **Exit Criteria**:
+    * **Feature complete** - The module owner declares the module feature complete for targeted use cases, and the API is ergonomic. 
+        >**Note**: Feature complete does not mean 100% AWS service feature coverage, instead, it means the module owner thinks we’ve implemented the minimum features required for a delightful developer experience for modeling the service infrastructure as code.
+     * **Successful API review** - conduct an API review/developer experience demo and invite relevant stakeholders
```

Comment:
Would be good to be more prescriptive about "relevant stakeholders" so we can consistently engage the right experts to be confident in the API's shape.

Reply:
Makes sense, I'll be more explicit. In general I'm thinking the relevant stakeholders are: 
- service's PM-T
- service's Developer Advocates (if any)
- SDE designated by service's leadership
- Solutions Architects (sourced from aws-cdk-interest or other email list)
- Developer Tools DAs